include/circt/Conversion/CalyxToFSM.h

```diff
@@ -29,10 +29,17 @@ namespace calyxToFSM {
 // Entry and exit state names of the Calyx control program in the FSM.
 static constexpr std::string_view sEntryStateName = "fsm_entry";
 static constexpr std::string_view sExitStateName = "fsm_exit";
+static constexpr std::string_view sGroupDoneInputs = "fsm_group_done_inputs";
```

Comment:
nit: I think this attribute names should technically have a `calyx.` prefix, since they are discardable attributes as opposed to inherent attributes of an FSM (where inherent attributes are not prefixed.)